### ISSUE TITLE
Displays not Showing

### ISSUE THREAD

**Issue by RedReaperLp** [SebLague/Digital-Logic-Sim#124]
![image](https://user-images.githubusercontent.com/64808428/210229323-7dd22f7b-85d6-4bbe-b407-165434b3cff2.png)
![image](https://user-images.githubusercontent.com/64808428/210229386-bb238906-6cac-4735-b4ad-286eac3a9690.png)

I Added the Outputs Later to be able to use it Anyways, but the Display Is not showing like in your Videos (I´m not sure about, if it is edited in your video)

BW

**Comment by SebLague** [comment]
Yeah sorry that feature isn't properly implemented yet, but will be added in a future update.

**Comment by RedReaperLp** [comment]
Ahh, okay, tyvm

**Comment by nathan-stewart** [comment]
Aha! My 10yo got really frustrated this morning trying to get this to work.

**Comment by y2k04** [comment]
Same, thanks for letting us know @SebLague!

**Comment by JaydenDev** [comment]
When will this be fixed? And is it possible to fix it my self atm?

**Comment by Highlandword9** [comment]
> ```@SebLague is there any version that let's you put the screen in the chip?```

@elogan321gamesalt I don't believe that there's a public version with that feature yet.

**Comment by y2k04** [comment]
> > `@SebLague is there any version that let's you put the screen in the chip?`
> 
> @elogan321gamesalt I don't believe that there's a public version with that feature yet.

Unfortunately no, which is why I created this issue because he told us that it wasn't ready: #144
